tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Requests/CreateOrderItemBaseRequest.cs

```diff
@@ -60,6 +65,10 @@ public void SetPayload(string key)
         }
         public void AssertPayload(OrderItemEntity actual)
         {
+            var timeUnit = Payload.CatalogueItemType == CatalogueItemType.AssociatedService
```

Comment:
Why check the type and set, and not just check string.isnullorwhitespace(Payload.TimeUnitName)?